systests/transports/pom.xml

```diff
@@ -331,6 +331,11 @@
             <version>${cxf.bcprov.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.vintage</groupId>
+            <artifactId>junit-vintage-engine</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Why the need for the vintage-engine dependency here?

Reply:
The `transports` uses a mix of JUnit 4 and JUnit 5 tests, we need the Vintage engine to run JUnit 4 tests using JUnit 5 platform, otherwise they are going to be skipped.